### PR TITLE
Start/End Dates in Export View should be in MM/DD/YYYY format.

### LINKED ISSUE #2314: Start/End Dates in Export View should be in MM/DD/YYYY format.
APD > Export and Submit > Continue to Review

"Start date - End date:" should display dates in MM-DD-YYYY format

### This task is done when...
 - [ ] displayed date format is correct

### PR DESCRIPTION
**Pull request by radavis**
Resolves #2314 

### This pull request changes...

- date format in export view

### This pull request is ready to merge when...

- [x] Tests have been updated (and all tests are passing)
- [x] This code has been reviewed by someone other than the original author
- [x] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented
- [x] The change has been documented
  - ~[ ] Associated OpenAPI documentation has been updated~
- [x] Changelog is updated as appropriate

### This feature is done when...

- [x] Design has approved the experience
- [x] Product has approved the experience

### CODE REVIEW

**Comment by mgwalker** [comment]
The activity dates here are `MM/DD/YYYY` instead of `MM-DD-YYYY`. A Jerome question, for sure, but from a code review perspective, 👍

**Comment by jeromeleecms** [comment]
@radavis @mgwalker Greg is correct, for consistency, let's keep it as MM/DD/YYYY.

File: web/src/containers/viewOnly/ApdStateProfile.js [hunk 1]
```diff
@@ -60,14 +60,14 @@ const ApdStateProfile = ({ stateProfile, keyPersonnel }) => {
...
-      <addr>
+      <address>
         {stateProfile.medicaidOffice.address1}
         <br />
         {stateProfile.medicaidOffice.address2 &&
           stateProfile.medicaidOffice.address2}
         {stateProfile.medicaidOffice.city}, {stateProfile.medicaidOffice.state}{' '}
         {stateProfile.medicaidOffice.zip}
-      </addr>
+      </address>
```
Comment (radavis) [inline web/src/containers/viewOnly/ApdStateProfile.js:70]: The console was complaining that `<addr>` was not a valid tag.

File: web/src/containers/viewOnly/activities/Activity.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ const Activity = ({ activity, activityIndex }) => {
               )
             )}
           </ul>
-        </p>
+        </div>
```
Comment (radavis) [inline web/src/containers/viewOnly/activities/Activity.js:30]: Nesting a `<ul>` within. `<p>` tag is frowned upon, apparently. Or, at least that's what the JS console says.

File: web/src/containers/viewOnly/ScheduleSummary.test.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  test('with no activities', () => {
+    const component = shallow(<ScheduleSummary activities={[]} />)
+    expect(component.text()).toMatch(/There are no milestones scheduled./)
+  });
+
+  test('with activities', () => {
+    const component = shallow(<ScheduleSummary activities={activities} />)
+    expect(component.text()).toMatch(/Program Administration/)
+  });
+});
```
Comment (radavis) [inline web/src/containers/viewOnly/ScheduleSummary.test.js:61]: Bonus test

**Pull request merged**